erpnext/accounts/doctype/pos_invoice/pos_invoice.py

```diff
@@ -161,7 +161,7 @@ def validate_pos_reserved_batch_qty(self, item):
 
 		bold_item_name = frappe.bold(item.item_name)
 		bold_extra_batch_qty_needed = frappe.bold(
-			abs(available_batch_qty - reserved_batch_qty - item.qty)
+			abs(available_batch_qty - reserved_batch_qty - item.qty * item.conversion_factor)
```

Comment:
Why not just use item.stock_qty instead?

Reply:
yeah that is also correct